src/components/Card/Card.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component, PropTypes } from 'react'
+import { View } from 'react-native'
+import Uranium from 'uranium'
+import { Elevation, Breakpoints, Colors, connectTheme } from '../../index'
+
+class Card extends Component {
+  render() {
+    const { style, children, ...other } = this.props
+    return (
+      <View
+        css={[
+          styles.root,
+          style,
+        ]}
+        {...other}>
+        {children}
+      </View>
+    )
+  }
+}
+
+Card.propTypes = {
+  children: PropTypes.node,
+  theme: PropTypes.object.isRequired,
+  style: PropTypes.oneOfType([
+    PropTypes.object,
+    PropTypes.array,
+  ]),
+}
+
+const styles = {
+  root: {
+    marginTop: 4,
+    marginBottom: 4,
+    marginLeft: 8,
+    marginRight: 8,
```

Comment:
Nit, margin vertical/horizontal, more succinct :P

Also, I like ordering css props a la csslint, so another nit is, like breaks between here and the flex stuff below. Groupings go:

positioning
box model (width/height -> padding -> border -> margin)
flexbox
text
background
other stuff

with line breaks between all of them.